conf/json-schema/README.md

```diff
@@ -0,0 +1,11 @@
+# the JSON Schema of JSON Schema
+
+This is downloaded from https://json-schema.org/draft-04/schema and modifies a line in order to support `bigint` type.
+
+```diff
+@@ -18,3 +18,3 @@
+          "simpleTypes": {
+-             "enum": [ "array", "boolean", "integer", "null", "number", "object", "string" ]
++             "enum": [ "array", "boolean", "integer", "null", "number", "object", "string", "bigint" ]
```

Comment:
Is upgrading ajv an option?

Reply:
No. The latest Ajv has not supported BigInt yet.

JSON doesn't have the minimum/maximum of `number` type, so minimum/maximum (and precision) are implementation-dependent.

Also, JSON Schema doesn't have the minimum/maximum of `number`/`integer` type by default. We can specify those with `minimum` and `maximum` properties. Therefore, `{ type: "integer" }` sounds like BigInt in JavaScript. Ajv may support a new option that allows both number and bigint values in `{ type: "integer" }` (https://github.com/epoberezkin/ajv/issues/1116). But I'm not sure if it's what we want.

Reply:
Thanks for this reply and for doing the associated research in ajv.

I think this means we're deciding that our rule schema is now officially an augmentation of the JSON Schema metaschema, which supports `"type": "bigint"`. Is that correct?

I feel that a change like this would require a TSC agenda item just to make sure we're okay making that change (unless this has already been discussed by TSC). Please let me know if you disagree.